etherpad 2.5.0 update broke

**Describe the bug**
After running the update to 2.5.0 etherpad fails to successfully start

<!--
NOTE: This Ansible playbook installs tens of separate services. If you're having a problem with a specific one, it is likely that the problem is with the service itself. You may wish to report that problem at the source, upstream.
-->

**To Reproduce**
My `vars.yml` file looks like this:

```yaml
etherpad_enabled: true
etherpad_admin_username: joe
etherpad_admin_password: <secret>
```

<!-- Below this line, tell us what you're doing to reproduce the problem. -->


**Expected behavior**
etherpad should start

**Matrix Server:**
 - OS: Ubuntu 24.04
 - Architecture [amd64]

**Additional context**
```
Sep 05 22:33:55 xwaretech.net systemd[1]: matrix-etherpad.service: Main process exited, code=exited, status=1/FAILURE
Sep 05 22:33:55 xwaretech.net systemd[1]: matrix-etherpad.service: Failed with result 'exit-code'.
Sep 05 22:34:25 xwaretech.net systemd[1]: matrix-etherpad.service: Scheduled restart job, restart counter is at 279.
Sep 05 22:34:25 xwaretech.net systemd[1]: Starting matrix-etherpad.service - Etherpad (matrix-etherpad)...
Sep 05 22:34:25 xwaretech.net matrix-etherpad[2460890]: 28084c0bb34fe3f50de14a379b84e4189efc267b33f68b92980bd1ef03725c8b
Sep 05 22:34:25 xwaretech.net systemd[1]: Started matrix-etherpad.service - Etherpad (matrix-etherpad).
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]: > etherpad@2.5.0 prod /opt/etherpad-lite
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]: > pnpm --filter ep_etherpad-lite run prod --settings /config/settings.json --credentials /data/credentials.json --apikey /data/apikey.json
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]: > ep_etherpad-lite@2.5.0 prod /opt/etherpad-lite/src
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]: > cross-env NODE_ENV=production node --require tsx/cjs node/server.ts --settings /config/settings.json --credentials /data/credentials.json --apikey /data/apikey.json
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]: /opt/etherpad-lite/src/node/utils/Cli.ts:41
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:     exports.argv.credentials = arg;
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:                  ^
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]: TypeError: Cannot set properties of undefined (setting 'credentials')
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:     at <anonymous> (/opt/etherpad-lite/src/node/utils/Cli.ts:41:18)
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:     at Object.<anonymous> (/opt/etherpad-lite/src/node/utils/Cli.ts:55:1)
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:     at Module._compile (node:internal/modules/cjs/loader:1688:14)
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:     at Object.transformer (/opt/etherpad-lite/node_modules/.pnpm/tsx@4.20.5/node_modules/tsx/dist/register-D46fvsV_.cjs:3:1104)
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:     at Module.load (node:internal/modules/cjs/loader:1423:32)
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:     at Function._load (node:internal/modules/cjs/loader:1246:12)
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:     at TracingChannel.traceSync (node:diagnostics_channel:322:14)
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:     at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:     at Module.require (node:internal/modules/cjs/loader:1445:12)
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:     at require (node:internal/modules/helpers:135:16)
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]: Node.js v22.18.0
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]: /opt/etherpad-lite/src:
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  ep_etherpad-lite@2.5.0 prod: `cross-env NODE_ENV=production node --require tsx/cjs node/server.ts --settings /config/settings.json --credentials /data/credentials.json --apikey /data/apikey.json`
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]: Exit status 1
Sep 05 22:34:26 xwaretech.net matrix-etherpad[2460925]:  ELIFECYCLE  Command failed with exit code 1.
Sep 05 22:34:27 xwaretech.net systemd[1]: matrix-etherpad.service: Main process exited, code=exited, status=1/FAILURE
Sep 05 22:34:27 xwaretech.net systemd[1]: matrix-etherpad.service: Failed with result 'exit-code'.
Sep 05 22:34:57 xwaretech.net systemd[1]: matrix-etherpad.service: Scheduled restart job, restart counter is at 280.
```

--- a/templates/systemd/etherpad.service.j2
+++ b/templates/systemd/etherpad.service.j2
@@ -35,15 +35,12 @@ ExecStartPre={{ devture_systemd_docker_base_host_command_docker }} create \
       --label-file={{ etherpad_base_path }}/labels \
       --health-interval={{ etherpad_container_health_interval }} \
       --mount type=bind,src={{ etherpad_config_path }},dst=/config,ro \
-      --mount type=bind,src={{ etherpad_data_path }},dst=/data \
       {% for arg in etherpad_container_extra_arguments %}
       {{ arg }} \
       {% endfor %}
       {{ etherpad_container_image_self_build_name if etherpad_container_image_self_build else etherpad_container_image }} \
       pnpm run prod \
-        --settings /config/settings.json \
-        --credentials /data/credentials.json \
-        --apikey /data/apikey.json
+        --settings /config/settings.json
 
 {% for network in etherpad_container_additional_networks %}
 ExecStartPre={{ devture_systemd_docker_base_host_command_docker }} network connect {{ network }} {{ etherpad_identifier }}

